src/lambda_function/s3_to_glue/events/generate_test_event.py

```diff
@@ -16,8 +16,8 @@
 import json
 import boto3
 
-SINGLE_RECORD_OUTFILE = 'single-record.json'
-MULTI_RECORD_OUTFILE = 'records.json'
+SINGLE_RECORD_OUTFILE = './src/lambda_function/s3_to_glue/events/single-record.json'
+MULTI_RECORD_OUTFILE = './src/lambda_function/s3_to_glue/events/records.json'
```

Comment:
I don't like this change, actually. We shouldn't assume that this script will always be run from the repository root. If we run it anywhere else (like in the script directory) we'll end up with this unnecessarily deep folder nesting (`/src/lambda_function/s3_to_glue/events/src/lambda_function/s3_to_glue/events/single-record.json` for example). Why don't we make the output directory a command line argument instead? Or leave it as it is and run this script from the script directory in `upload-and-deploy` so that we can avoid the awkward `sam local invoke -e ../../..` syntax?

Reply:
Yeah the problem I ran into was that we had be inside the directory of the lambda app prior to running that command of `sam local invoke`, so it forced the `records.json` to be generated within that directory under `events`. 

I am in favor of the latter. This way for github actions, we'll be running the script from the repository root and if we need to run it outside of GH actions, we can specify where to save it.